src/ssr-carousel-track.vue

```diff
@@ -198,14 +198,14 @@ export default
 	&.dragging
 		pointer-events none
 
+	// Apply direction property which was reset in .ssr-carousel
+	&.ssr-carousel-rtl 
+		direction rtl
+
```

Comment:
I would have thought this would mess up the layout of the slides on the track.  Also, the carousel looks broke on the deploy preview: https://deploy-preview-140--vue-ssr-carousel.netlify.app/accessibility/

![vue-ssr-carousel demo 2024-02-05 at 4 23 23 PM jpg](https://github.com/BKWLD/vue-ssr-carousel/assets/77567/87d59524-7d5b-420e-a146-d7c014864177)